### ISSUE TITLE
calculator - funkcje pomocnicze

### ISSUE THREAD

**Issue by justina1224** [daft-academy/daftacademy-python4beginners-autumn2021#19]
Czy w ostatnim zadaniu z kalkulatorem dodatkowe funkcje również nie były dozwolone ? testy przeszly mi dopiero jak przerzucilam ta funkcje (showResult) jako funkcje wewnetrzna.. 

![image](https://user-images.githubusercontent.com/40146662/143957964-3be9089f-2005-40f1-aa08-e870687d92f9.png)

**Comment by krwiozercza-szynszyla** [comment]
@justina1224 tak, jest test sprawdzający czy w module nie ma zadeklarowanych nadmiarowych `identifiers` poza funkcją `calculator`
